pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/LoadManager.java

```diff
@@ -87,4 +97,14 @@
     public void disableBroker() throws Exception;
 
     public void stop() throws PulsarServerException;
+
+    static LoadManager create(final ServiceConfiguration conf, final PulsarService pulsar) {
+        switch (conf.getLoadManagerName()) {
+            case "NewLoadManager":
```

Comment:
I have used a reflective approach to assume a constructor with a `PulsarService` argument exists. I could also use registration of classes if that is more desirable.